serverless/serverless.yml

```diff
@@ -115,51 +71,41 @@ functions:
       - httpApi:
           path: /pokemon
           method: post
-  update:
-    handler: ./src/handler.update
-    events:
-      - httpApi:
-          path: /pokemon/{id}
-          method: patch
-  remove:
-    handler: ./src/handler.remove
-    events:
-      - httpApi:
-          path: /pokemon/{id}
-          method: delete
   import:
     handler: ./src/handler.importPokemon
     events:
       - httpApi:
           path: /pokemon/import
           method: post
-  search:
-    handler: ./src/handler.search
-    events:
-      - httpApi:
-          path: /pokemon/search
-          method: get
-  featured:
-    handler: ./src/handler.featured
-    events:
-      - httpApi:
-          path: /pokemon/featured
-          method: get
-  imageDownloader:
-    handler: ./src/handler.imageDownloader
+  worker:
+    handler: ./src/handler.worker
     events:
       - sqs:
           arn:
             Fn::GetAtt:
               - AssetsQueue
               - Arn
 
+  # update:
```

Comment:
Do we want to keep this?

Reply:
Yeah, I want to keep them in case we want to add them back in the future